apps/admin-gui/src/app/facilities/pages/resource-detail-page/resource-tags/resource-tags.component.ts

```diff
@@ -0,0 +1,140 @@
+  displayedColumns = [];
+
+  addAuth: boolean;
+  removeAuth: boolean;
+  editAuth: boolean;
+
+  ngOnInit() {
+    this.pageSize = this.tableConfigService.getTablePageSize(this.tableId);
+
+    this.route.parent.params.subscribe(params => {
+      this.voId = params['voId'];
+      this.resourceId = params['resourceId'];
+      this.resourcesManager.getResourceById(this.resourceId).subscribe(resource => {
+        this.resource = resource;
+        this.updateData();
+      })
+    });
+  }
+
+  removeTags() {
+    const config = getDefaultDialogConfig();
+    config.width = '450px';
+    config.data = {items: this.selection.selected.map(tag => tag.tagName),
+      title: 'RESOURCE_DETAIL.TAGS.REMOVE_TAGS_DIALOG_TITLE',
+      description: 'RESOURCE_DETAIL.TAGS.REMOVE_TAGS_DIALOG_DESCRIPTION',
+      theme: 'resource-theme'};
+
+    const dialogRef = this.dialog.open(UniversalRemoveItemsDialogComponent, config);
+
+    dialogRef.afterClosed().subscribe(result => {
+      if (result) {
+        this.removeTag(this.selection.selected);
+      }
+    });
+  }
+
+  removeTag(tags: ResourceTag[]) {
+    if(tags.length === 0) {
+      this.notificator.showSuccess(this.translate.instant('RESOURCE_DETAIL.TAGS.REMOVED_SUCCESSFULLY'));
+      return this.updateData();
+    }
+    const tag = tags.pop();
+    this.resourcesManager.removeResourceTagFromResource({
+      resource: this.resourceId,
+      resourceTag: tag
+    }).subscribe(() => {
+      this.removeTag(tags);
+    });
+  }
+
+  addTag() {
+    const config = getDefaultDialogConfig();
+    config.width = '600px';
+    config.data = {voId: this.voId, resourceId: this.resourceId, assignedTags: this.resourceTags, theme: 'resource-theme'};
+
+    const dialogRef = this.dialog.open(AddResourceTagToResourceDialogComponent, config);
+
+    dialogRef.afterClosed().subscribe( success => {
+      if (success) {
+        this.notificator.showSuccess(this.translate.instant('RESOURCE_DETAIL.TAGS.ADDED_SUCCESSFULLY'));
+        this.updateData();
+      }
+    });
+  }
+
+  updateData() {
+    this.loading = true;
+    this.selection.clear();
+    this.resourcesManager.getAllResourcesTagsForResource(this.resourceId).subscribe(tags => {
+      this.resourceTags = tags;
+      this.selection.clear();
+      this.setAuthRights();
+      this.loading = false;
+    });
+  }
+
+  setAuthRights() {
+    this.displayedColumns = [];
+
+    // TODO: edit auth objects
+    this.addAuth = this.authResolver.isAuthorized('assignResourceTagToResource_ResourceTag_Resource_policy', [this.resource]);
+    this.removeAuth = this.authResolver.isAuthorized('removeResourceTagFromResource_ResourceTag_Resource_policy', [this.resource]);
+    this.editAuth = this.authResolver.isAuthorized('updateResourceTag_ResourceTag_policy', [this.resource]);
+
+    this.displayedColumns = this.removeAuth ? ['select', 'id', 'name'] : ['id', 'name'];
+
+    if(this.editAuth){
+      this.displayedColumns.push('edit');
+    }
```

Comment:
About edit column in assigned tags on resource: I dont think that it harms anything if its there but I found it personally counfusing honestly. Functionally this page is for adding/removing tags from resource and editing the actual tag name here is.. not what I would expect and I think the casual user can be confused. 
I would personally dont put this column here

Reply:
Edit column has been removed.